privx_api/response.py

```diff
@@ -97,6 +97,14 @@ def __init__(
     def __str__(self) -> str:
         return "PrivXStreamResponse {}".format(self.status)
 
+    def _format_content(self) -> Union[str, bytes]:
```

Comment:
See above comment. This is not needed and would make things more complex. Calling this function also has a side effect of not closing the response connection in the PrivXStreamResponse, Delegate the processing of the stream data to the consumer of the PrivXStreamResponse. Instead, focus on delivering a unified interface.